### PR TITLE
🧪testing out the new system

### PR DESCRIPTION
**Pull request by jaredpar2**
what type of comment is needed?

### CODE REVIEW

**Comment by sharwell** [comment]
Interesting that the integration tests get skipped for compiler changes

File: src/Compilers/Core/Portable/ConstantValue.cs [hunk 1]
```diff
@@ -11,6 +11,7 @@
 
 namespace Microsoft.CodeAnalysis
 {
+    // jaredpar was here
```
Comment (davidwengier) [inline src/Compilers/Core/Portable/ConstantValue.cs:14]: Nit:
```suggestion
    // jaredpar2 was here
```

Comment (jaredpar) [reply src/Compilers/Core/Portable/ConstantValue.cs:14]: who is the real jaredpar?

**Comment by jaredpar** [comment]
okay, so need explicit azp comment to kick off build. what happens when I push again

**Comment by jaredpar2** [comment]
looks like on push the pipelines don't auto-rerun. need a fresh comment

**Pull request closed without merging**